### PR TITLE
DAT-4563: Add tests for Content Review

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
and complete comeng training ;)

cc @Wikia/west-wing @ludwikkazmierczak 

see also: https://github.com/Wikia/selenium-config/pull/24

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialJsPage.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special;
+
+import org.openqa.selenium.WebDriver;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialJsPage.java]: MINOR Remove this unused import 'org.openqa.selenium.WebDriver'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AceEditor.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject;
+
+import org.openqa.selenium.WebDriver;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AceEditor.java]: MINOR Remove this unused import 'org.openqa.selenium.WebDriver'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ContentReviewModule.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            return false;
+        }
+    }
+
+    public boolean isSubmitLinkVisible() {
+        boolean isLinkVisible = false;
+        try {
+            wait.forElementVisible(submitForReviewLink, 3, 1);
+            isLinkVisible = true;
+        } catch(TimeoutException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ContentReviewModule.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ContentReviewModule.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ContentReviewModule.java]: MINOR Remove this unused import 'org.openqa.selenium.WebDriver'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ContentReviewModule.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+    public ContentReviewModule() {
+        super();
+    }
+
+    public boolean isModuleVisible(){
+        try {
+            wait.forElementVisible(contentReviewModule, 5, 1);
+            return true;
+        }catch (TimeoutException e){
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ContentReviewModule.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues:
* MAJOR 2 major
* MINOR 3 minor

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialJsPage.java [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+import org.openqa.selenium.support.FindBy;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.TestContext;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ContentReviewModule;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+/**
+ * Created by Ludwik on 2015-09-25.
```
Comment (SebastianMarzjan) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialJsPage.java]: High time to merge ;-)

**Comment by SebastianMarzjan** [comment]
If they pass, LGTM

**Pull request merged**